aiida/cmdline/commands/cmd_computer.py

```diff
@@ -411,36 +413,57 @@ def computer_list(all_entries, raw):
 @with_dbenv()
 def computer_show(computer):
     """Show detailed information for a computer."""
-    echo.echo(computer.full_text_info)
+    table = []
+    table.append(['Label', computer.label])
+    table.append(['PK', computer.pk])
+    table.append(['UUID', computer.uuid])
+    table.append(['Description', computer.description])
+    table.append(['Hostname', computer.hostname])
+    table.append(['Transport type', computer.transport_type])
+    table.append(['Scheduler type', computer.scheduler_type])
+    table.append(['Work directory', computer.get_workdir()])
+    table.append(['Shebang', computer.get_shebang()])
+    table.append(['Mpirun command', ' '.join(computer.get_mpirun_command())])
+    table.append(['Prepend text', computer.get_prepend_text()])
+    table.append(['Append text', computer.get_append_text()])
+    echo.echo(tabulate.tabulate(table))
 
 
 @verdi_computer.command('rename')
 @arguments.COMPUTER()
 @arguments.LABEL('NEW_NAME')
+@deprecated_command("This command has been deprecated. Please use 'verdi computer relabel' instead.")
```

Comment:
Completely unrelated, but should we maybe have `@deprecated_command` automatically append the "This command has been deprecated" text to the string passed through, so it is not necessary to include that part each time?

Reply:
That would certainly be an option, although low priority I would say. Usually you copy this line from some other place anyway so it is already there.